l10n_it_fiscal_document_type/tests/test_doc_type.py

```diff
@@ -80,3 +89,59 @@ def test_doc_type_refund(self):
             invoice.journal_id.id
         )
         self.assertEqual(refund.fiscal_document_type_id.id, self.TD04.id)
+
+    @users("Adviser", "admin")
+    def test_access(self):
+        """Users can only read fiscal documents,
+        Users can't create/update/delete fiscal documents."""
```

Comment:
Quindi l'utente root può comunque creare/scrivere/cancellare?

Reply:
Mi aspetterei di sì perché OdooBot se ne infischia delle regole di accesso.
Per sicurezza ho provato in runboat ed effettivamente è così: 
![image](https://github.com/OCA/l10n-italy/assets/101628000/70420fbd-945a-44ea-a363-cb0fa686c34e)

Reply:
Ottimo, grazie.